src/guide/running-multiple-simulations/index.rst

```diff
@@ -202,6 +204,7 @@ Long Argument                  Short Argument              Description
                                                            By default the :enum:`ErrorLevel<flamegpu::CUDAEnsemble::EnsembleConfig::ErrorLevel>` will be set to "slow" (1).
 ``--standby``                                              Allow the operating system to enter standby during ensemble execution.
                                                            The standby blocking feature is currently only supported on Windows, where it is enabled by default.
+``--no-mpi``                                               Do not use MPI (only available when built with ``FLAMEGPU_ENABLE_MPI`` at CMake configuration time).
```

Comment:
I've not yet reviewed the main PR, but not sure why this would be neccesary, as the way to not use mpi is to not use mpirun to launch the program? in which case the comm world would be 1? 

This might be one of the awkward buts of having optional MPI though, and I should review the actual code first really.